nss-day-cohort-39/tabloid-full-stack-velours
# my profile link and styling

### Changes
- Added a "My Profile" link in header bar
- Converted Categories, Reactions, Tags, and Users into an Admin dropdown in the navbar
- Added thumbnails to users list
- Uploaded profile images for all users
- Replaced dummy data in posts for presentation purposes

### Testing
- Make sure My Profile link works
- Make sure links in dropdown in header all work
